### DISCUSSION TITLE
Music-videos - Naming script

### DISCUSSION THREAD

**Discussion by Yosh82** [RandomNinjaAtk/docker-lidarr-extended#73] [Q&A] [answered]
Hi. Does anyone know if it's possible to edit the script which names the video files? I've had a look at the script but could see where it does the rename.

Example:

Currently, it names it "ArtistName\Belong to the World-video.mkv"

I'd like to change this to:
ArtistName\ArtistName - Belong to the World-video.mkv"

Any help would be appreciated.

Thanks in advance.

**Comment by RandomNinjaAtk** [comment, marked as answer]
The script, does not support any changes to file naming. What is the use-case for this? Other than personal preference.

**Reply by Yosh82** [reply]
Thanks for the response. It was purely personal preference, makes it easier to search in Plex with the artist name in the filename, but I know it should really be accessed via the music library with the global music-videos path which works fine.

I've been using lidarr for many years now, but this -extended release is a game changer. Tagging is actually better than the original for some reason even though both use musicbrainz, but the tidal dl's are amazing!

Thank you @RandomNinjaAtk - Top job!

**Reply by RandomNinjaAtk** [reply]
I don't believe it's possible to search for a music video in Plex... Thanks for the compliments.

**Reply by Yosh82** [reply]
I created a custom video library which I planned to search for the artist, which is when I realised it could only search by the file name. It's not a great solution anyway, but I was using a different naming convention at the time. Now I've set L-E to do it all, its much easier to leave as is with the global music-video path set.

Thanks again!

**Reply by RandomNinjaAtk** [reply]
Emby and Kodi both handle music videos better, if you haven't given them a try, you should.

**Reply by Yosh82** [reply]
Thanks, I've used both many years ago, but Plex is easier for most users who connect and on most smart TV's, but PlexAmp is the main reason I love plex (plus I've had a lifetime membership for a while too now). Sonic analyst is very good and they seem to keep adding features. 
In my opinion, plexamp analysis and gui is better than tidal, spotify etc. My music library is around 150,000 songs and handles it amazing well.

Also, have to note, I can use Plexamp on Android auto in my car too 👍
